src/openforms/formio/serializers.py

```diff
@@ -41,13 +41,13 @@ def apply_hidden_state(
         Hidden fields in Formio don't run *any* validation, see the
         ``Component.shouldSkipValidation`` method for reference.
         """
-        config_wrapper = FormioConfigurationWrapper(configuration)
-
         # initial_data is only set if Serializer(data=...) was used, and we can't take
         # (dynamic) hidden state into account without initial data.
         if not hasattr(self, "initial_data"):
             return
 
+        config_wrapper = FormioConfigurationWrapper(configuration)
+
```

Comment:
Is this only a performance improvement?

Reply:
yeah it is, I noticed it when stepping through it and forgot to revert the change for this fix